HIP/hip-0000-token-update-remove-keys.md

```diff
@@ -0,0 +1,183 @@
+---
+hip: 0000
+title: Remove Existing Keys From A Token
+author: Justyn Spooner <justyn@dovu.io>
+working-group: Justyn Spooner (@justynjj), Cooper Kunz (@cooper_kunz), Jason Fabritz (@bugbytesinc)
+type: Standards Track
+category: Application
```

Comment:
```suggestion
type: Informational
```

Reply:
Shouldn't this be a Standard - Service change?

Reply:
Adding from the discussion you created for this HIP:
The intent here with my Informational suggestion was to have a standard with a "trash key" that SDKs could adopt. I was trying to steer it in this direction because the actual key removal may have unintended consequences. Take, for instance, a KYC key scenario. A contract could be using KYC and a disgruntled employee who used to have access to that key could remove it and mess up the original contract. There may be legal issues with this case, but I'm not a lawyer.

There was a further conversation that updated keys may need to sign the tx, so you don't ever really fix the problem with a trash key. In any case, I would like to note this in this discussion.